src/Provider/FreeGeoIp/Readme.md

```diff
@@ -10,6 +10,24 @@
 This is the Free GeoIp provider from the PHP Geocoder. This is a **READ ONLY** repository. See the
 [main repo](https://github.com/geocoder-php/Geocoder) for information and documentation. 
 
+# Freegeoip Shutdown
+As per the [freegeoip.net](http://freegeoip.net/shutdown) website, the provider has been purchased by [IpStack](https://ipstack.com/).
+As a result, this provider no longer works with the default configuration. It will still work if you use the 
+[self hosted variant](https://github.com/apilayer/freegeoip/) and supply a host when constructing the provider.
+
```

Comment:
I think we should also add a link to https://github.com/geocoder-php/ipstack-provider :)

Reply:
Updated :)